setup/IaC/modules/automationaccount.bicep

```diff
@@ -316,6 +316,15 @@ resource module12 'modules' = if (newDeployment || updatePSModules) {
       }
     }
   }
+  resource module30 'modules' = if (newDeployment || updatePSModules) {
```

Comment:
@dutt0 module30 is already being used by Ali in his PR which is why I had used module31, think you will have to make module 32

Reply:
Yes @singhgss Just kept it as a placeholder. I still need to update the FR msgs. I will resolve the conflict after merging all of your branches before I merge to main.